.circleci/config.yml

```diff
@@ -106,6 +106,45 @@ jobs:
     steps:
       - build-steps
 
+  build-macos:
+      macos:
+        xcode: 11.6.0
+      steps:
+       - checkout-all
+       - run:
+          name: get conda
+          command: |
+            brew install wget
+            wget https://repo.anaconda.com/archive/Anaconda3-2021.11-MacOSX-x86_64.sh -O ~/anaconda.sh
+            bash ~/anaconda.sh -b -p $HOME/anaconda
+            source ~/anaconda/bin/activate
+            conda init zsh
+            source ~/.zshrc
+            conda env create -f ./conda/macos.yml
+       - run:
+          name: build
+          command: |
+            source ~/.zshrc
+            conda activate vecsim
+            make
+       - run:
+          name: unit test
+          command: |
+            source ~/.zshrc
+            conda activate vecsim
+            make unit_test
+       - run:
+          name: flow test
+          command: |
+            source ~/.zshrc
```

Comment:
Let's add this to a script or make target -then you can call it directly. WDYT

Reply:
We already have a script: https://github.com/RedisLabsModules/readies/blob/master/bin/getconda
Just need to integrate the logic there: we can have `FULL=1` or `MINI=1` selectors to get the Anaconda we require.
Note that macOS/M1 should also be in mind (not for CircleCI but we shouldn't write things that work only in CircleCI anyway).